packages/shady-css-scoped-element/package.json

```
@@ -23,6 +23,7 @@
   "license": "BSD-3-Clause",
   "scripts": {
     "build": "gulp",
+    "build:watch": "chokidar 'src/**/*.js' -c 'npm run build'",
```

```suggestion
    "build:watch": "chokidar --initial 'src/**/*.js' -c 'npm run build'",
```
(guessing based on the others)